patches/nginx-1.19.9-privileged_agent_process.patch

```diff
@@ -188,7 +192,7 @@ index 7cee1c5..c4f70d6 100644
 +static void
  ngx_cache_manager_process_handler(ngx_event_t *ev)
  {
-     time_t        next, n;
+     ngx_uint_t    i;
```

Comment:
why change this ?

Reply:
may be the previous patch is not base on 1.19.9